tests/input.csv

```diff
@@ -1,2 +1,2 @@
-sample,gvcf,vcf_path,vcf_index_path
-chr22,false,https://github.com/nf-core/test-datasets/raw/refs/heads/modules/data/genomics/homo_sapiens/illumina/vcf/NA12878.chr22.1X.vcf.gz,
+sample,label,gvcf,vcf_path,vcf_index_path
+chr22,chr22,false,https://github.com/nf-core/test-datasets/raw/refs/heads/modules/data/genomics/homo_sapiens/illumina/vcf/NA12878.chr22.1X.vcf.gz,
```

Comment:
I thought in nf-core they changed to making that link, at least the test repos, see e.g.https://github.com/nf-core/ampliseq/blob/8f139cecd95a23920617f2c2b154733494ed4562/conf/test.config#L28-L29. Not sure if that makes sense to do that here as well. probably not.